frontend/src/pages/questions/ViewQuestion.tsx

```diff
@@ -46,31 +47,39 @@ const ViewQuestion: React.FC = () => {
   }
 
   return (
-    <Box width="100%" height="85vh" my={5}>
+    <Box width="100%" height="100vh" my={5}>
       <Allotment>
         <Allotment.Pane>
-          <VStack as="div" style={{ overflowY: 'auto', height: '100%', padding: '16px' }}>
-            <Heading as="h1" size="xl" textAlign="center">
-              {question.title}
-            </Heading>
-            <Text fontSize="md" color={colourScheme} mt={2}>
-              <span style={{ fontWeight: 'bold' }}>Complexity: </span>
-              <QuestionComplexityTag questionComplexity={question.complexity} />
-            </Text>
-            <Text fontSize="md" color={colourScheme} mt={2}>
-              <span style={{ fontWeight: 'bold' }}>Categories:</span> {question.categories.join(', ')}
-            </Text>
-            <Text fontSize="md" color={colourScheme} mt={2}>
-              <span style={{ fontWeight: 'bold' }}>Link to Question: </span>
-              <Link href={question.linkToQuestion}>{question.linkToQuestion}</Link>
-            </Text>
-            <Divider mt={4} />
-            <VStack align="start" spacing={4} mt={4}>
-              <Heading as="h2" size="md">
-                Description
+          <VStack as="div" alignItems="start" style={{ overflowY: 'auto', height: '100%', padding: '16px' }}>
+            <VStack alignSelf="center" alignItems="center">
+              <Heading as="h1" size="xl" textAlign="center">
+                {question.title}
               </Heading>
-              <Text whiteSpace="pre-line">{question.questionDescription}</Text>
+              <Text fontSize="md" color={colourScheme} mt={2}>
+                <span style={{ fontWeight: 'bold' }}>Complexity: </span>
+                <QuestionComplexityTag questionComplexity={question.complexity} />
+              </Text>
+              <Text fontSize="md" color={colourScheme} mt={2}>
+                <span style={{ fontWeight: 'bold' }}>Categories:</span> {question.categories.join(', ')}
+              </Text>
+              <Text fontSize="md" color={colourScheme} mt={2}>
+                <span style={{ fontWeight: 'bold' }}>Link to Question: </span>
+                <Link href={question.linkToQuestion}>{question.linkToQuestion}</Link>
+              </Text>
             </VStack>
+            <Divider mt={4} />
+            <Box ml="8px">
+              <VStack align="start" spacing={4} mt={4}>
+                <Heading as="h2" size="md">
+                  Description
+                </Heading>
+                <Text
+                  dangerouslySetInnerHTML={{
+                    __html: DOMPurify.sanitize(question.questionDescription),
+                  }}
+                />
```

Comment:
wrt to the comment above

```suggestion
                <Text
                  whiteSpace="pre-line"
                  dangerouslySetInnerHTML={{
                    __html: DOMPurify.sanitize(question.questionDescription),
                  }}
                />
```

| `whiteSpace="pre-line"` |  without `whiteSpace="pre-line"`  |
|-|-|
| <img width="1437" alt="Screenshot 2023-09-20 at 11 43 53 PM" src="https://github.com/CS3219-AY2324S1/ay2324s1-course-assessment-g17/assets/79991214/ef7c5b76-b41e-44b8-b168-31d56629e47f"> | <img width="1413" alt="Screenshot 2023-09-20 at 11 44 08 PM" src="https://github.com/CS3219-AY2324S1/ay2324s1-course-assessment-g17/assets/79991214/f1214940-58ba-4f63-bcbb-db002d140ca0"> |

Reply:
(oh this might be fixed in the prev commit)